centreon/www/front_src/src/AdditionalConnectorConfiguration/Listing/Listing.tsx

```diff
@@ -0,0 +1,55 @@
+import { MemoizedListing } from '@centreon/ui';
+
+import ActionsBar from './ActionsBar/ActionsBar';
+import useListing from './useListing';
+import DeleteConnectorDialog from './Columns/Actions/DeleteDialog';
+import useColumns from './Columns/columns';
+
+const Listing = (): JSX.Element => {
+  const { columns } = useColumns();
+
+  const {
+    changePage,
+    changeSort,
+    page,
+    resetColumns,
+    setLimit,
+    selectColumns,
+    sortf,
+    sorto,
+    isLoading,
+    data,
+    selectedColumnIds,
+    openEditDialog
+  } = useListing();
+
+  return (
+    <>
+      <MemoizedListing
+        actions={<ActionsBar />}
+        columnConfiguration={{
+          selectedColumnIds,
+          sortable: true
+        }}
+        columns={columns}
+        currentPage={(page || 1) - 1}
```

Comment:
Is it possible to start the page from 0 instead of 1 then decrease the page manually?